docs/production-deployment/cloud/metrics/reference.mdx

```diff
@@ -26,146 +26,282 @@ keywords:
   - gRPC error count
 ---
 
-Most Temporal Cloud metrics are suffixed with `_count`.
-This indicates that they behave largely like a [Prometheus counter](https://prometheus.io/docs/concepts/metric_types/#counter).
-You'll want to use a function like `rate` or `increase` to calculate a per-second rate of increase, or an extrapolated total increase over a time period.
+A metric is a measurement or data point that provides insights into the performance and health of a system.
+This document describes the metrics available on the Temporal Cloud platform.
+Temporal Cloud metrics help you monitor performance and troubleshoot errors.
+They provide insights into different aspects of the Service.
+
+This guide offers an overview of:
+
+* **[Metrics capture](#capture)**:
+  Gather metrics using a Prometheus-style endpoint so they can be visualized using Prometheus and Grafana or exported to observability platforms like Datadog.
+* **[Available Temporal Cloud metrics](#available-metrics)**:
+  The metrics emitted by Temporal Cloud include counts of gRPC errors, requests, successful task matches to a poller, and more.
+* **[Metrics labels](#metrics-labels)**:
+  Temporal Cloud metrics labels can filter metrics and help categorize and differentiate results.
+* **[Operations](#metrics-operations)**:
+  An operation is a special type of label that categorizes the type of operation being performed when the metric was collected.
+
+:::info SDK METRICS
+
+This document discusses metrics emitted by [Temporal Cloud](/cloud).
+Temporal SDKs also emit metrics, sourced from Temporal Clients and Worker processes.
+You can find information about Temporal SDK metrics on its [dedicated page](/references/sdk-metrics).
+
+Please note:
+
+* SDK metrics start with the phrase `temporal_`.
+  These metrics are distinct from Temporal Cloud metrics.
+* Temporal Cloud metrics start with `temporal_cloud_`.
+  They are documented here.
+
+:::
+
+## Metrics capture {#capture}
```

Comment:
Changing to Capture metrics, but it still feels like it can be confused with being a modifier of the noun "metrics". Maybe if I put Temporal Cloud in there it would work. Checking out the style guide. Good catch.

Reply:
I ended up with "Gather". Thanks again.